lua/caramba/refactor.lua

```
@@ -24,64 +24,31 @@ M.operations = {
 -- Rename symbol with AI assistance
 function M.rename_symbol(new_name, opts)
   opts = opts or {}
-  
+  local ts_utils = require("nvim-treesitter.ts_utils")
```

Luacheck (1.2.0)

[warning] shadowing upvalue 'ts_utils' on line 8

(W431)